server/src/main/java/org/elasticsearch/action/ingest/SimulateIndexResponse.java

```diff
@@ -31,13 +31,15 @@ public class SimulateIndexResponse extends IndexResponse {
     private final BytesReference source;
     private final XContentType sourceXContentType;
 
+    @SuppressWarnings("this-escape")
```

Comment:
Unrelated?

Reply:
It will go away when we merge main, but it is breaking my local testing.